bluebrain/repo-patches/packages/highfive/package.py

```diff
@@ -15,6 +15,7 @@ class Highfive(CMakePackage):
     git = "https://github.com/BlueBrain/HighFive.git"
 
     version("master", branch="master")
+    version("2.8.0", commit="4535cd8279c814c49e4c73b83e24ee164dfc7ae3")
```

Comment:
I assume this is a test, and the commit is going to be replaced by a sha

Reply:
Yes sorry. This is to test that it does not fail. After we will generate the new version.